frontend/src/pages/admin/Template/sections/Environment.vue

```diff
@@ -25,22 +25,23 @@
                             :error="item.error"
                             :disabled="item.encrypted"
                             value-empty-text=""
-                            :type="(!readOnly && (editTemplate || item.policy === undefined))?'text':'uneditable'"></FormRow>
-                        <!-- <ff-text-input  v-model="item.name" :disabled="item.encrypted"  /> -->
+                            :type="(!readOnly && (editTemplate || item.policy === undefined)) ? 'text' : 'uneditable'"
+                        ></FormRow>
                     </td>
-                    <td class="px-4 py-4 border w-auto align-top" :class="{'align-middle':item.encrypted}">
-                        <div class="w-full" v-if="!item.encrypted">
```

Comment:
I believe we'll still want the `item.encrypted` feature to show the padlock icon. The `settings['team:environment-variable-view']` can be added as an extra fork the if statement